guides/tutorials/codelabs/azure-vmss-source-to-prod/index.md

```diff
@@ -0,0 +1,329 @@
+---
+layout: single
+title:  "Azure VM Scale Sets: Source to Prod"
+sidebar:
+  nav: guides
+---
+
+{% include toc %}
+
+In this codelab, you will create a source-to-prod continuous delivery pipeline for a simple java based application targeting VM Scale Sets in Azure.
+
+After completing this lab the following workflow will happen: 
+  1. A developer pushes a commit to Github
+  2. Jenkins automatically builds a Debian package and pushes it to an Aptly repository
+  3. Spinnake bakes the new package into an image
+  4. Spinnaker deploys to a test environment and disable the previous deployment
+  4. Spinnake waits for a manual judgement of the test environment
+  5. Spinnaker deploys the image to the production environment and disable the previous deployment.
+  
+This codelab will walk through the following steps to implement this scenario: 
+
+  * [Deploy Spinnaker and Jenkins on Azure](#1-deploy-spinnaker-and-jenkins-on-azure)
+  * [Verify Jenkins](#2-verify-jenkins)
+  * [Prepare your environments](#3-prepare-your-environments)
+  * [Create the Continuous Delivery pipeline](#4-create-the-continuous-delivery-pipeline)
+  * [Trigger your pipeline](#5-trigger-your-pipeline)
+  * [Clean up the environment](#6-tear-down)
+
+
+## 1. Deploy Spinnaker and Jenkins on Azure 
+
+You need the following resources to perform the codelab: 
+- **Azure subscription**: You can create one for free using the [Azure Free trial](https://azure.microsoft.com/free/).
+- **Spinnaker and Jenkins VM**:
+
+From a shell run the following command that will walk you through all the steps necessary to deploy Jenkins and Spinnaker in Azure.
+
+> NOTE: Be careful to select `vmss` when prompted so when you run the following script.

```

Comment:
"when prompted so when you run" seems weird and I'm not sure what the phrasing is supposed to be. Maybe "Be careful to select `vmss` when prompted by the following script."